src/deterministic_brb.rs

```diff
@@ -280,7 +307,20 @@ impl<A: Actor<S>, SA: SigningActor<A, S>, S: Sig, BRBDT: BRBDataType<A>>
         };
 
         info!("[BRB] {} initiating bft for msg {:?}", self.actor(), msg);
-        self.broadcast(&Payload::BRB(Op::RequestValidation { msg }), self.peers()?)
+        let (mut self_packets, mut others_packets): (Vec<_>, Vec<_>) = self
```

Comment:
I think  we need a comment here or at top of func about short-circuit logic / purpose.